config/api.yml

```diff
@@ -742,6 +742,25 @@
       :post:
       - :name: assign
       - :name: unassign
+  :firmwares:
+    :description: Firmwares
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
@abellotti @imtayadeway @rodneyhbrown7 I removed the firmwares subcollection option for now. If this PR gets merged before the linked PR (#14028), the linked PR will be updated to add this as a subcollection.